DropDown/src/DropDown.swift

```diff
@@ -635,16 +657,13 @@ extension DropDown {
 	}
 	
 	private func fittingWidth() -> CGFloat {
-		struct Static {
-			static let templateCell = DropDownCell.Nib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
-		}
-		
+		let templateCell = cellNib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
```

Comment:
Yes, we could! I have seen this method is not called so often, so I did not spend time doing that, but if you think is needed we can...